### ISSUE TITLE
Save should be placed somewhere else

### ISSUE THREAD

**Issue by ChloeWolfieGirl** [ubports/gallery-app#19]
In the gallery app, the save button is where the back button should be.
Perhaps put SAVE to the top right like it is on android.
This leaves the question of undo and redo.

Perhaps we should get rid of the revert button as tapping the back button would mean no changes would be saved, and put undo and redo at the bottom where that used to be.

**Comment by TronFourtyTwo** [comment]
Yes, the editing UI and behaviour need a refresh, at the moment is quite confusing.

**Comment by StefWe** [comment]
As first and quick step we can change the description of the save button from **Edit Photo** to **Save edit photo**
I think this makes the function of the button much easier to understand.
